src/test/consensus/Consensus_test.cpp

```diff
@@ -93,6 +94,7 @@ class Consensus_test : public beast::unit_test::suite
         using namespace csf;
         using namespace std::chrono;
 
+        testcase("slow peer");
```

Comment:
I actually added them back because I thought that was `rippled` style.  If that isn't the case, my other squelching changes aren't so important ;).